core/src/main/java/org/apache/accumulo/core/conf/AccumuloConfiguration.java

```diff
@@ -71,18 +72,22 @@ private static class PrefixProps {
    * Gets a property value from this configuration.
    *
    * <p>
-   * Note: this is inefficient, but convenient on occasion. For retrieving multiple properties, use
-   * {@link #getProperties(Map, String...)} if the property names are known or
-   * {@link #getProperties(Map, Predicate)} with a custom filter.
+   * Note: this is inefficient for values that are not a {@link Property}. For retrieving multiple
+   * properties, use {@link #getProperties(Map, Predicate)} with a custom filter.
    *
    * @param property
    *          property to get
    * @return property value
    */
   public String get(String property) {
-    Map<String,String> propMap = new HashMap<>(1);
-    getProperties(propMap, property);
-    return propMap.get(property);
+    try {
+      return get(Property.valueOf(property));
```

Comment:
The try portion of this block is always going to throw, because we would only ever call it with the property key, which is never going to be the same as the enum key, which is what `.valueOf` is trying to look up.

Reply:
Great catch, I totally missed that. I should be calling `Property.getPropertyByKey` instead?

Reply:
Would be nice to have a test for this that would fail before any changes

Reply:
How would you test it if it falls back to filtering against all properties?

Reply:
> Great catch, I totally missed that. I should be calling `Property.getPropertyByKey` instead?

If we have such a method, then yeah, that would probably work.

Reply:
I see #2876